GameServerCore/IGame.cs

```diff
@@ -25,7 +25,7 @@ public interface IGame: IUpdate
         /// <summary>
         /// Interface for all map properties used for the game.
```

Comment:
Update summary `Contains all map related game settings such as collision handler, navigation grid, announcer events, and map properties. Doubles as a Handler/Manager for all MapScripts.`